dali/test/python/test_operator_random_bbox_crop.py

```diff
@@ -366,7 +368,8 @@ def test_random_bbox_crop_no_labels():
     test_box_shape = [200, 4]
 
     def get_boxes():
-        out = [(np.random.randint(0, 255, size=test_box_shape, dtype=np.uint8) / 255).astype(dtype=np.float32) for _ in range(batch_size)]
+        out = [(np.random.randint(0, 255, size=test_box_shape, dtype=np.uint8) /
+                255).astype(dtype=np.float32) for _ in range(batch_size)]
```

Comment:
The `/` sign at the end of this line might be confused with `\` as line break. If you feel, that it might be an issue, maybe you can consider breaking this line in some other manner?